config/locales/en.yml

```diff
@@ -8,6 +8,8 @@ en:
   solidus_subscriptions:
     subscription:
       actionable_date: "Actionable Date"
+    currency:
+      invalid_code: "is not a valid currency code"
```

Comment:
Maybe it's better to move this error message [here](https://github.com/solidusio-contrib/solidus_subscriptions/blob/master/config/locales/en.yml#L107-L116)